# BRCA
All data generated or analyzed during this study are included in this published article and its supplementary information files. The bioinformatics components of this study were derived from publicly accessible data platforms and are publicly accessible with these URL (no accession number needed), e.g., (1) GEPIA2: http://gepia2.cancer-pku.cn/#index; (2) UALCAN: https://ualcan.path.uab.edu/index.html; (3) GSCA: https://guolab.wchscu.cn/GSCA/#/; (4) TIMER: https://compbio.cn/timer3/; (5) STRING: https://cn.string-db.org/; (6) STITCH: https://stitch-db.org/cgi/input.pl?UserId=ea98VI4LTqas&sessionId=xNgDS2ENv2JP&input_page_show_search=on; (7) Single cell portal: https://singlecell.broadinstitute.org/single_cell; (8) GDSC: https://www.cancerrxgene.org/; (9) CTRP: https://portals.broadinstitute.org/ctrp/.
Using the Single-cell portal online server (https://singlecell.broadinstitute.org/single_cell), disease=breast cancer was selected, and datasets that could support subgroup analyses were manually screened, resulting in the following data from the study by Wu et al. (https://singlecell.broadinstitute.org/single_cell/study/SCP1039/a-single-cell-and-spatially-resolved-atlas-of-human-breast-cancers?genes=CXCL2&cluster=Entire%20Breast%20Cancer%20Atlas%20-%20All%20cells&spatialGroups=--&annotation=celltype_minor--group--study&(subsample=100000&tab=distribution#study-visualise). Single-cell Portal identifier was Study SCP1039 and the GEO Series accession number was GSE176078. The expression of CXCL2 in immune cells was verified. Subgroups TNBC, ER+, and HER2+ were selected for analysis.
The GWAS outcome factors for the Two-sample Mendelian randomization (TSMR) analysis were sourced from the FinnGen database (https://www.finngen.fi/en/access_results) with access number of “C3_BREAST_EXALLC”, “C3_BREAST_ERPLUS_EXALLC”, “C3_BREAST_ERNEG_EXALLC”, “CD2_INSITU_BREAST_INTRADUCTAL_EXALLC”, “CD2_INSITU_BREAST_LOBULAR_EXALLC”, and “CD2_INSITU_BREAST_EXALLC”. The exposure factors, including immune cells, were obtained from the GWAS Catalog (https://www.ebi.ac.uk/gwas/, access number of PMID 32929287, searching in GWAS Catalog could access these original data), and genetic instruments were from the MRInstruments R package (access URL: https://github.com/MRCIEU/MRInstruments). All accession numbers and associated platforms have been fully released and are available for verification.
All integrated analysis results generated through platforms such as GEPIA 2, UALCAN, and GSCA have been provided via supplementary data files or figures within the main text to enable direct verification of the specific expression patterns of the identified biomarkers.
